ui/view/Settings.qml

```diff
@@ -56,8 +56,18 @@ ColumnLayout {
                 color: Style.content_secondary
                 //: settings tab, version label
                 //% "v"
-                text: qsTrId("settings-version") + " " + viewModel.version
+                text: qsTrId("settings-version") + viewModel.version
+                
+                MouseArea {
+                anchors.fill:parent
+                acceptedButtons: Qt.LeftButton
+                cursorShape: Qt.PointingHandCursor
+                onClicked: function () {
+                    BeamGlobals.copyToClipboard(qsTrId("settings-version") + viewModel.version)
```

Comment:
I think you can refer to the control's .text field, this allows you to avoid duplication  of `qsTrId("settings-version") + viewModel.version`